docs/CHANGELOG.asciidoc

```diff
@@ -199,6 +199,8 @@ coming[6.3.1]
 
 Reduce the number of object allocations made by {security} when resolving the indices and aliases for a request ({pull}30180[#30180])
 
+Stop setting connection request timeout in default low-level REST client configuration ({pull}30384[#30384])
```

Comment:
Maybe add "because it causes timeout exceptions before any requests are made which isn't how timeouts are supposed to work" or something like that.